### ISSUE TITLE
Multi Speaker 迁移学习

### ISSUE THREAD

**Issue by xukai98** [ming024/FastSpeech2#65]
你好，关于multi speaker的tts是基于迁移学习的吗

**Comment by ming024** [comment]
@xukai98 No I just train the model from scratch. It's just a simple demo.

**Comment by xukai98** [comment]
Thanks for your reply.
Is each speaker trained individually?
Or are all the speakers trained together?

**Comment by ming024** [comment]
@xukai98 Yes, they are trained together. There is a speaker embedding table containing the speaker representations of all the speakers. And this embedding table, is end-to-end trained, together with the TTS model.

**Comment by Tian14267** [comment]
请问你有做迁移学习吗？我使用作者的预训练模型做迁移训练，合成出来的语音没有声音。请问迁移训练需要调整哪些地方？ @xukai98 @ming024

**Comment by mortyzhou77** [comment]
> 请问你有做迁移学习吗？我使用作者的预训练模型做迁移训练，合成出来的语音没有声音。请问迁移训练需要调整哪些地方？ @xukai98 @ming024

请问您解决了这个问题吗
